scripts/cms-any/merge-ups.php

```diff
@@ -59,6 +59,9 @@
     rename_file_if_exists('.github/workflows/merge-ups.yml', '.github/workflows/merge-up.yml');
 }
 
-if (!module_is_recipe() && !is_meta_repo()) {
-  write_file_even_if_exists('.github/workflows/merge-up.yml', $content);
+if (!module_is_recipe() && !current_branch_is_non_semver()) {
```

Comment:
Why do we care if the branch is non-semver here?

Reply:
Have changed a little to `if (current_branch_is_semver() && !module_is_recipe()) {`

Reason being that if we're processing a standard semver style branch we can assume that there's multiple major version supported because that's our convention.  For things without semver style branches (main/master) assume there's no need to merge-up anything